### PR TITLE
Update to rails 6.1.4

### PR DESCRIPTION
**Pull request by modullar**

### CODE REVIEW

File: soapy_cake.gemspec [hunk 1]
```diff
@@ -7,19 +7,20 @@ require 'soapy_cake/version'
...
+  spec.homepage      = 'http://combostrike.com'
   spec.license       = 'MIT'
 
   spec.files         = `git ls-files -z`.split("\x0")
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'activesupport'
+  spec.add_dependency 'activesupport', '~> 6.1.4'
```
Comment (tsujigiri) [inline soapy_cake.gemspec:23]: How about `~> 6.1`, instead, to make it less restrictive?

Comment (modullar) [reply soapy_cake.gemspec:23]: I intended the restriction. I want everything to be coherent and clear.

Comment (tsujigiri) [reply soapy_cake.gemspec:23]: This is a gem, though, which may or may not be used by others, who may or may not use other versions of `activesupport`.

Comment (modullar) [reply soapy_cake.gemspec]: I agree. I think anyway we have to upgrade the projects where this library is being used to at least `6.1`, which wouldn't do much. I will specify the version to be greater or equal to 6.1.

**Review by tsujigiri — APPROVED** [review]

**Pull request merged**